assets/javascript/modal.js

```diff
@@ -0,0 +1,44 @@
+const modalElement = document.getElementById('modal');
+const modalOverlays = document.querySelectorAll('.modal--overlay');
+const closeBtns = document.querySelectorAll('.modal-close');
+
+function openModal() {
+  if(modalElement.classList) {
+    modalElement.classList.add('open');
+  }
+}
+
+function closeModal() {
+  modalElement.classList.remove('open');
+}
+
+function customizeModal(developer) {
+  document.getElementById('modal--title').innerHTML = developer.name;
+  document.getElementById('modal--body--icon').innerHTML = '<img alt="' + developer.name + '" class="developer" src="assets/images/employees/' + developer.id + '.png">';
+  document.getElementById('modal--body--description').innerHTML = developer.description;
+  let buffs = '<div class="row">\n' +
```

Comment:
try to keep a standard, you had used single quotes here to variable and before in another block of HTML you didn't